pac-aurora-provisioner/cloudformation/aurora-pac-master.yml

```diff
@@ -145,10 +154,36 @@ Resources:
                 - Key: name
                   Value: !Sub ${TagName} - ${EnvironmentName}
 
+    DBClusterFromSnapshot:
```

Comment:
Example 3 [here](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-conditions.html) seems to do this in the same block - did that not work for you?

Reply:
That example works with `AWS::RDS::DBInstance` but it doesn't for `AWS::RDS::DBCluster`. :S

Reply:
I just tried with the following template:
```
  DBCluster:
        Type: AWS::RDS::DBCluster
        Properties:
            Engine: aurora
            DatabaseName: !Ref DBInstanceDBName
            DBSubnetGroupName: !Ref DBSubnetGroup
            VpcSecurityGroupIds: !Ref DBSecurityGroups
            DBClusterParameterGroupName: !Ref DBClusterParameterGroup
            BackupRetentionPeriod: !Ref DBBackupRetentionPeriod
            MasterUsername: !If [UseDBSnapshot, !Ref "AWS::NoValue", !Ref DBInstanceMasterUsername]
            MasterUserPassword: !If [UseDBSnapshot, !Ref "AWS::NoValue", !Ref DBInstanceMasterPassword]
            SnapshotIdentifier: !If [UseDBSnapshot, !Ref SourceSnapshot, !Ref "AWS::NoValue"]
            Tags:
                - Key: environment
                  Value: !Ref TagEnvironment
                - Key: teamDL
                  Value: !Ref TagTeamDL
                - Key: systemCode
                  Value: !Ref TagSystemCode
                - Key: ipCode
                  Value: !Ref TagIpCode
                - Key: description
                  Value: !Ref TagDescription
                - Key: name
                  Value: !Sub ${TagName} - ${EnvironmentName}
```
and it worked for me... What did you try?